.github/CODEOWNERS

```diff
@@ -50,6 +52,7 @@ yarn.lock                                           @backstage/community-plugins
 /workspaces/report-portal                           @backstage/community-plugins-maintainers  @yashoswalyo
 /workspaces/rollbar                                 @backstage/community-plugins-maintainers  @andrewthauer
 /workspaces/scaffolder-backend-module-annotator     @backstage/community-plugins-maintainers  @BethGriggs
+/workspaces/scaffolder-backend-module-kubernetes    @backstage/community-plugins-maintainers  @BethGriggs
```

Comment:
@BethGriggs is this safe to assume you own this?

Reply:
@awanlin @BethGriggs can you add @debsmita1 to annotator and kubernetes. Maybe others... Thanks a lot. Otherwise I can raise another PR for annotator of course.

Reply:
Sure, I can do that, won't be until tomorrow or the weekend though.

Reply:
Sorry, this clearly was not done when I said it would be. 😞